<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfKeyBindingsWidget</class>
 <widget class="QWidget" name="WkfKeyBindingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>快捷键</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>选定命令的快捷键:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QKeySequenceEdit" name="keySequenceEdit"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="resetButton">
            <property name="toolTip">
             <string extracomment="Reset selected shortcut to default"/>
            </property>
            <property name="text">
             <string>重置</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearButton">
            <property name="toolTip">
             <string extracomment="Clear selected shortcut"/>
            </property>
            <property name="text">
             <string>清除</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="assignButton">
            <property name="toolTip">
             <string extracomment="Apply shortcut to selected command"/>
            </property>
            <property name="text">
             <string>分配</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
